ccmlib/cluster.py

```diff
@@ -536,11 +533,15 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=True,
             time.sleep(2)  # waiting 2 seconds to check for early errors and for the pid to be set
         else:
             for node, p, mark in started:
+                if not node._wait_for_running(p, timeout_s=7):
```

Comment:
is if we have say 3 nodes, we will wait up to 7 seconds for each in a sequence?

Reply:
yes, but it does not matter...
because if they are started in parallel... then after we catch 1st running we will with high probability immediately have others running as well.